package/patterns-yast.spec

```diff
@@ -146,7 +146,10 @@ Recommends:     yast2-iscsi-client
 Recommends:     yast2-journal
 Recommends:     yast2-ldap-client
 Recommends:     yast2-nfs-client
+# YaST NIS packages are dropped from TW (bsc#1183893)
+%if 0%{?suse_version} > 1500
```

Comment:
Just for reference for future readers, https://en.opensuse.org/openSUSE:Packaging_for_Leap#RPM_Distro_Version_Macros

Reply:
BTW, did you mean _sle\_version_ instead of _suse\_version_?

According to the link pasted above, _suse\_version_ is 1599 for openSUSE TW. What am I missing?

Reply:
Ah, there is an inconsistency in https://en.opensuse.org/openSUSE:Build_Service_cross_distribution_howto#Detect_a_distribution_flavor_for_special_code 

Anyway, fixed to use the `sle_version`.